ctapipe/reco/FitGammaHillas.py

```diff
@@ -0,0 +1,308 @@
+from itertools import combinations,permutations
+
+import numpy as np
+
+from scipy.optimize import minimize
+
+from astropy import units as u
+u.dimless = u.dimensionless_unscaled
+
+from ctapipe.utils import linalg
+
+from ctapipe.reco.reco_algorithms import RecoShowerGeomAlgorithm
+from ctapipe.io.containers import RecoShowerGeom
+
+
+
+__all__ = ["FitGammaHillas"]
+
+
+
+
+def guessPixDirection(pix_x, pix_y, tel_phi, tel_theta, tel_foclen, camera_rotation=-100.893 * u.degree):
+    # TODO replace with proper implementation
+    # beta is the pixel's angular distance to the centre according to beta / tel_view = r / maxR
+    # alpha is the polar angle between the y-axis and the pixel
+    # to find the direction the pixel is looking at, 
+    #  - the pixel direction is set to the telescope direction
+    #  - offset by beta towards up
+    #  - rotated around the telescope direction by the angle alpha
+    
+    pix_alpha = np.arctan2(pix_x, pix_y)
+    pix_beta  = (pix_x**2 + pix_y**2)**.5
+
+    pix_beta  = pix_beta / tel_foclen * u.rad 
```

Comment:
perhaps mark it as deprecated so there is no confusion.  So far we have no specific decorator for that, but you can use the astropy one for now:
```python
from astropy.utils.decorators import deprecated

@deprecated(0.1, "will be replaced with real coord transform")
def func():
    ...
```